crates/sui-graphql-rpc/src/types/transaction_block_effects.rs

```diff
@@ -346,13 +333,25 @@ impl ObjectChange {
                     .data_unchecked::<PgManager>()
                     .fetch_obj(sui_address, Some(version.value()))
                     .await?;
-                Ok(Some(Self {
+                Ok(Self {
                     input_state: None,
                     output_state,
                     id_created: None,
                     id_deleted: None,
-                }))
+                })
             }
         }
     }
 }
+
+impl TryFrom<StoredTransaction> for TransactionBlockEffects {
+    type Error = Error;
+
+    fn try_from(stored: StoredTransaction) -> Result<Self, Error> {
+        let native = bcs::from_bytes(&stored.raw_effects).map_err(|e| {
```

Comment:
The compiler does not complain about needing to specify the type for `native`?

Reply:
It is inferred from the fact that it's a field of `TransactionBlockEffects`.